build.gradle

```diff
@@ -5,6 +5,7 @@ plugins {
     id 'maven-publish'
     id 'checkstyle'
     id 'com.github.johnrengelman.shadow' version '7.1.2'
+    id "com.modrinth.minotaur" version "2.+"
```

Comment:
i think exact version is better

Reply:
From Modrinth:
> It's safest to have this on 2.+ to get the latest features and bug fixes without having to worry about breaking changes.

Reply:
That is if you trust people code actually don't break stuff hehe

Reply:
I think we can trust Modrinth xD